backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/common/utils/ansible/AnsibleRunnerHttpClient.java

```diff
@@ -88,6 +89,7 @@ public AnsibleReturnValue artifactHandler(UUID uuid, int lastEventID, int timeou
             }
             lastEventID = processEvents(uuid.toString(), lastEventID, fn, "", Paths.get(""));
             iteration += POLL_INTERVAL / 1000;
+            Thread.sleep(POLL_INTERVAL);
```

Comment:
1. how about avoiding redundant sleep here with something like:
```java
             lastEventID = processEvents(uuid.toString(), lastEventID, fn, "", Paths.get(""));
             if (lastEventID == -1) {
                 return returnValue;
             }
            iteration += POLL_INTERVAL / 1000;
             if (iteration > timeout * 60) {
                 // Cancel playbook, and raise exception in case timeout occur:
                 cancelPlaybook(uuid, timeout);
                 throw new TimeoutException(
                         "Play execution has reached timeout");
             }
             Thread.sleep(POLL_INTERVAL);
```

2. please remove the comment at line 79 and move the comment in line 80 to the call to processEvents

3. how about remaing `iteration` to `executionTime`?